tests/routes/test_items.py

```diff
@@ -535,9 +535,9 @@ def test_items_geometry_return_options(app):
     assert len(body["features"]) == 1
     assert body["numberMatched"] == 1
     assert body["numberReturned"] == 1
-    assert body["features"][0]["id"] == "1"
+    assert body["features"][0]["id"] == 1
     assert body["features"][0]["properties"]["ogc_fid"] == 1
-    assert "geometry" not in body["features"][0]
+    assert not body["features"][0]["geometry"]
```

Comment:
We do not exclude `None` so the dict has `geometry=None` (which is still a valid geojson)